tests/components/smartthings/test_sensor.py

```diff
@@ -75,6 +78,84 @@ async def test_entity_three_axis_invalid_state(hass, device_factory):
     assert state.state == STATE_UNKNOWN
 
 
+async def test_entity_power_consumption_report_state(hass, device_factory):
+    """Test the state attributes."""
+    test_attr_values = {
+        "start": "00110501T000000-0800",  # Nov 5 2020 America/Los Angeles
+        "end": "00110601T000000-0800",  # Nov 6 2020 America/Los Angeles
+        "energy": 10000,
+        "power": 90,
+        "deltaEnergy": 100,
+        "powerEnergy": 80,
+        "energySaved": 300,
+        "persistedEnergy": 20000,
+    }
+    device = device_factory(
+        "Power Consumption Report",
+        [Capability.power_consumption_report],
+        {Attribute.power_consumption: test_attr_values},
+    )
+    await setup_platform(hass, SENSOR_DOMAIN, devices=[device])
+    state = hass.states.get("sensor.power_consumption_report_start_time")
+    assert state.state == "00110501T000000-0800"
```

Comment:
The device class sensors should return a state that is an isoformatted datetime string in UTC time.